cypress/e2e/group2/myworkflows.ts

```diff
@@ -52,8 +52,15 @@ describe('Dockstore my workflows', () => {
 
     cy.visit('/my-workflows');
     cy.get('[data-cy=myWorkflowsMoreActionButtons]').should('be.visible').click();
+    cy.fixture('myWorkflows.json').then((json) => {
+      cy.intercept('GET', '/api/users/1/workflows', {
+        body: json,
+        statusCode: 200,
+      }).as('getWorkflows');
+    });
```

Comment:
Since the UI now issues a new GET request for the user's entries after clicking "Discover existing", these stubbed requests were added for the integration tests to work. I suppose the tests' main purpose is to check that the correct endpoint (workflows vs notebooks vs containers/appTools) is being called.